NeoML/test/src/ParameterLayerTest.cpp

```diff
@@ -0,0 +1,79 @@
+/* Copyright � 2017-2020 ABBYY Production LLC
```

Comment:
New format of 1st line copyright is
/* Copyright © < Year-created >-< Year-updated > ABBYY
So for new files
/* Copyright © 2024 ABBYY

All files should be saved in UTF8 codepage.